guide/webpack.config.js

```diff
@@ -32,6 +33,15 @@ module.exports = validate({
                 include: RegExp(__dirname + "/src"),
                 loaders:["react-hot", "babel-loader"]
             },
+	    {
+		test: /\.txt$/,
+		loader: 'raw-loader',
+		include: path.resolve(__dirname, 'src/examples/raw-code'),
+	    },
+	    {
+		test: /\.md$/,
+		loader: 'raw-loader',
+	    },
```

Comment:
Testing for `.txt` or `.md` isn't necessary for this use case.  It was taken from the loader's docs but I figured if we wanted to render say markdown this is the behaviour we would want. 

You might be right about the syntax being legacy. I was having trouble with the syntax from the docs and found that example in MUI's source. I will dig into this a little more.